dev-util/flawfinder/flawfinder-2.0.11.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{6,7} )
```

Comment:
Should 2.7 be dropped already? 

I'll add 3.8 in a separate commit since it seemed to pass test phase.

Reply:
I'll drop Python 2.7 today.

See: https://gentoo.org/support/news-items/2020-02-07-python-2-7-eol.html

Reply:
Looks like you merged it already. Should the 2.7 removal and 3.8 addition be in a single commit?

Reply:
I think I added python3.8 already, but generally I think that's okay if you have no revdeps. Then splitting commits would be better to revert any broken changes.